.github/workflows/ci.yml

```diff
@@ -203,11 +185,8 @@ jobs:
         submodules: recursive
 
     - name: Install nightly
```

Comment:
this naming doesn't seem right(?), but I left it as I found it. Let me know if this needs changing.